### DISCUSSION TITLE
Recurring job is always run ~15 seconds too early

### DISCUSSION THREAD

**Discussion by alexgutjahr** [jobrunr/jobrunr#494] [Q&A] [answered]
I've just started using JobRunr and gave it a shot with a recurring job using `JobScheduler.scheduleRecurrently(Cron.daily(4))`

I expect this to run every day at 4am, but according to the logs it runs a few seconds earlier:

```
- 2022-06-14T03:59:45.879000+0000
- 2022-06-15T03:59:46.320000+0000
```

This behavior is pretty consistent; I assume I'm just missing something and would appreciate any hints, thanks!

**Comment by rdehuyss** [comment, marked as answer]
This is noted in the documentation for recurring jobs - see https://www.jobrunr.io/en/documentation/background-methods/recurring-jobs/. I'm looking into fixing this but I think that exact scheduling will be part of the pro version.

**Reply by alexgutjahr** [reply]
Thanks @rdehuyss - I indeed did not read that part!
